command/agent/command.go

```diff
@@ -1407,6 +1422,10 @@ Client Options:
     Mark this node as a member of a node-class. This can be used to label
     similar node types.
 
+  -node-pool
+    Register this node under this node pool. If the node pool does not exist it
```

Comment:
```suggestion
    Register this node under this node pool. If the node pool does not exist it
```

Maybe s/under/in ? I think of stuff being _in_ a pool, rather than under it